### ISSUE TITLE
Fix arclight-demo deploy

### ISSUE THREAD

**Issue by anarchivist** [sul-dlss/arclight-demo#312]
The public ArcLight demo has been down for some time. Given that we've got some upcoming work on ArcLight this fall for Nuremberg, it's important that we have it up and running so @marlo-longley, @ggeisler, and @laurensorensen have a baseline to consult as we do our analysis. Ideally, this should be up and running by the week of LDCY (8/8).

**Comment by cbeer** [comment]
My memory is we need to completely redo how the app is deployed in AWS because they've removed or deprecated some feature or stack that we were using.

**Comment by marlo-longley** [comment]
Just an update here, after talking with @laurensorensen and info from Chris, we decided that the [sul-arclight-demo](https://sul-arclight-demo.stanford.edu/) page is sufficient for the purpose of demonstrating current Arclight functionality. This is the corresponding [repo](https://github.com/sul-dlss/stanford-arclight). This Stanford demo has custom theming and potentially settings/config (vs. a plain deploy of Arclight core using the demo template). But since the Stanford demo uses the latest Arclight core gem, it is fine for the purpose of showcasing features for the upcoming NTA workcycle.

tl;dr - working with AWS to get this running will be very labor intensive so we will not prioritize this ticket. sul-arclight-demo is sufficient for now.
